applications/machine_learning/configuration/thingy52_nrf52832/app_ZRelease.conf

```diff
@@ -25,6 +25,9 @@ CONFIG_ML_APP_ML_STATE_CONTROL_BUTTON_ID=0x0000
 
 CONFIG_CAF=y
 
+CONFIG_CAF_POWER_MANAGER=y
+CONFIG_CAF_KEEP_ALIVE_EVENTS=y
```

Comment:
I think we should document introducing power management in README (could be done in separate PR)